src/progress-bar-icon.js

```diff
@@ -48,16 +48,16 @@ export default function ProgressBarIcon() {
 							xlinkHref="#a3HIePHoOc"
```

Comment:
Changes in this file are outside the scope of this PR, why are they here?

Reply:
I applied these changes as I noticed that they're causing an error in the console. To avoid this error from happening, I applied this fix. Shall I revert it and create a separate PR for it?

Reply:
It's okay, however, in the future, keep those changes separate.

Reply:
I removed  that part from this PR and created a separate PR for it.